Unix/installbuilder/conf/omiserver.conf

```diff
@@ -20,6 +20,18 @@ httpsport=0
 ##
 #trace=(true|false)
 
+## To enable NTLM authentication via negotiation (SPNEGO), specify the location 
+## of the credentials file. This file is in the form
+##
+## <domain>|<username>:<password> 
+##
+## For further information, please see link: https://github.com/Microsoft/omi/blob/master/Unix/doc/setup-ntlm-omi.md
+##
+##
+##NtlmCredsFile=PATH
+## For example
+#NtlmCredsFile=/etc/opt/omi/.creds/ntlm
```

Comment:
I think you missed my prior comment. Please delete lines 30-32, you don't need them.